app/controllers/application_controller.rb

```diff
@@ -1 +1,4 @@
-class ApplicationController < ActionController::Base; end
+class ApplicationController < ActionController::Base
+  include ParamsLogging
+  before_action :add_params_to_request_store
```

Comment:
I've renamed this to LogRequestParams, to make it clear it should only be included in controllers, where a request context is guaranteed.

A middleware is a great idea and I started exploring it for a bit but I am not sure it's appropriate, as it would be a middleware depending on another middleware and harder to maintain. With the before_action part of the module now including LogRequestParams to a controller is not much work, and there may be cases we want to suppress the logging of query params entirely in some controllers.